src/scripts/itemDiscuss/dimItemDiscuss.directive.template.html

```diff
@@ -0,0 +1,100 @@
+<div id="discuss-drawer" ng-if="vm.show">
+    <div class="discuss-options">
+        <button class="dim-button" style="float: right;" ng-click="vm.cancel()" translate="Loadouts.Cancel"></button>
+    </div>
+
+    <div class="user-review-form">
+        <div class="user-review-community-overview">
+            <div ng-if="vm.item.communityReviews.length">
+                {{ vm.item.name }} {{ 'DtrReview.HasAverage' | translate }} {{ vm.item.dtrRating }} {{ 'DtrReview.Based' | translate }} {{ vm.item.communityReviews.length }} {{ 'DtrReview.ReviewCount' | translate }}
+            </div>
+            <div ng-if="!vm.item.communityReviews.length">
+                {{ vm.item.name }} {{ 'DtrReview.NoReviewsYet' | translate }}
```

Comment:
again for the sake of grammar
this should be changed to
`{{itemname}} has no community reviews yet. Be the first!`
with translate-values="{itemname: vm.item.name}"